app/documents/label_pdf.rb

```diff
@@ -41,7 +41,7 @@ def render_sample_details(sample)
     text_lines [
       "I.N. #{sample.isolate_name}".truncate(22),
       "I.M. #{sample.inactivation_method}".truncate(22),
-      "P.D. #{sample.date_produced == "Blinded" ? "Blinded" : sample.date_produced.strftime("%m/%d/%Y")}",
+      "P.D. #{sample.blinded_attribute?(:date_produced) ? sample.date_produced : sample.date_produced.strftime("%m/%d/%Y")}",
```

Comment:
Sorry Julien :( 
I didn't understood what you meant first, I do things fast because I thought you were suggesting to do what I did. Now I think is fixed. Thanks!